msteams-platform/bots/how-to/format-your-bot-messages.md

```diff
@@ -10,6 +10,9 @@ ms.author: anclear
 
 Message formatting enables you to bring out the best in bot messages. You can format your bot messages to include rich cards that are attachments that contain interactive elements, such as buttons, text, images, audio, video, and so on.
 
+> [!NOTE]
+> The bot message size limit is 50 KB. You can't send bot message if its size exceeds 50 KB.
```

Comment:
Please get this reviewed by the bots team.

We shouldn't say "bot message", it's not a type of message.

Reply:
And we also list message size limits on another page, - https://docs.microsoft.com/en-us/microsoftteams/limits-specifications-teams

Let's update that too.

Reply:
Followed up internally on this. Please document the message size limit at 40 KB.